Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -344,8 +349,47 @@ public async Task<TeamViewModel> UpdateTeamAsync(int teamID, TeamPatchViewModel
 
             return t;
         }
+
+        private async Task SendInviteEmail(int teamID, string inviteeUsername, string inviterUsername)
+        {
+            var team = _context.Team.FirstOrDefault(t => t.ID == teamID);
+            var activity = _context.Activity.FirstOrDefault(a => a.ID == team.ActivityID);
+            var invitee = _accountService.GetAccountByUsername(inviteeUsername);
+            var inviter = _accountService.GetAccountByUsername(inviterUsername);
+
+            string from_email = _config["Emails:Sender:Username"];
```

Comment:
Did you want to use a different address for sending recim notifications?

Reply:
> Did you want to use a different address for sending recim notifications?

I just mentioned this to Cam, but we could make the argument for either adding the "automated" tag to the recim account or we could get a 360recim email.